lib/matplotlib/image.py

```diff
@@ -1188,10 +1188,12 @@ def get_extent(self):
             raise RuntimeError('Must set data first')
         return self._Ax[0], self._Ax[-1], self._Ay[0], self._Ay[-1]
 
-    def set_filternorm(self, s):
+    @_api.rename_parameter("3.8", "s", "filternorm")
+    def set_filternorm(self, filternorm):
```

Comment:
these two seem a little pointless to add a rename to, considering they literally just pass, but still...